ALVRClient/Renderer.swift

```diff
@@ -371,13 +423,20 @@ class Renderer {
                 hudMessageBuffer.deallocate()
             case ALVR_EVENT_STREAMING_STARTED.rawValue:
                 print("streaming started: \(alvrEvent.STREAMING_STARTED)")
-                let foveationVars = FFR.calculateFoveationVars(alvrEvent.STREAMING_STARTED)
+                let settingsJsonBuffer = UnsafeMutableBufferPointer<CChar>.allocate(capacity: 1024)
+                alvr_get_settings_json(settingsJsonBuffer.baseAddress)
```

Comment:
This is unsafe. There needs to be some way of asking the ALVR client core how big the buffer needs to be, or have ALVR allocate the buffer itself and then transfer ownership to Swift.

Reply:
My bad, i forgot in my review. The usage is wrong. alvr_get_settings_json() requires first calling with nil buffer to get the bugger size, then call again to fill the buffer.